### DISCUSSION TITLE
Detecting if someone stands in front of door - from Top down with esp32cam

### DISCUSSION THREAD

**Discussion by dpunzeng** [jomjol/AI-on-the-edge-device#1993] [Q&A]
Hi There, I know its off-topic but I thought I might get a hint here: 
Does anyone think its possible to load a model that detects any single person standing in front of a door on a ESP32cam. The cam looks from ceiling down. I know that this can be done with better hw and eg movidus chip...
The question is, would the esp32cam board be powerfull enough for such a basic model.
Any input welcome,
Thx heaps

**Comment by uwen70** [comment]
Hi dpunzeng,
an ESP32Cam is able to detect Human und Cat Faces and to recognice Human Faces. There are libarys from espressif for deep learning: https://github.com/espressif/esp-dl.
Look at this project https://randomnerdtutorials.com/esp32-cam-video-streaming-face-recognition-arduino-ide/.
